test_driver/mocks/auth_service_mocks.dart

```diff
@@ -2,8 +2,12 @@ import 'dart:async';
 
```

Comment:
made sign in streams update the onAuthStateChanged stream with a signed in user

Reply:
The FakeAuthService is a boss.